Sources/Xcodeproj/pbxproj().swift

```diff
@@ -128,59 +133,64 @@ func xcodeProject(
 
     // Add a file reference for the package manifest itself.
     project.mainGroup.addFileReference(path: "Package.swift", fileType: "sourcecode.swift")
-    
+
     // Add a group for the overriding .xcconfig file, if we have one.
     let xcconfigOverridesFileRef: Xcode.FileReference?
     if let xcconfigPath = options.xcconfigOverrides {
+        // Verify that the xcconfig override file exists
+        if !exists(xcconfigPath) {
```

Comment:
(Which uses whatever filesystem is passed into this method, i.e. local or in memory, the global method uses local)